playbooks/provisioning/openstack/README.md

```diff
@@ -41,12 +41,23 @@ etc.). The result is an environment ready for openshift-ansible.
 Pay special attention to the values in the first paragraph -- these
 will depend on your OpenStack environment.
 
-The `env_id` and `openstack_dns_domain` will form the DNS domain all
+The `env_id` and `public_dns_domain` will form the cluster's DNS domain all
 your servers will be under. With the default values, this will be
-`openshift.example.com`.
-
-`openstack_nameservers` is a list of DNS servers accessible from all
-the created Nova servers. These will be serve as your DNS forwarders.
+`openshift.example.com`. For workloads, the default subdomain is 'apps'.
+
+The `public_dns_nameservers` is a list of DNS servers accessible from all
+the created Nova servers. These will be serving as your DNS forwarders for
+external FQDNs that do not belong to the cluster's DNS domain.
+
+The `openshift_use_dnsmasq` controls either dnsmasq is deployed or not.
```

Comment:
It's not clear why you may want to **not** use `dnsmasq` ... I believe there are some additional upcoming changes around DNS, so IMHO we should just leave it as-is and don't support not deploying without dnsmasq. Can you please elaborate why we should allow to deploy without?